addons/google_calendar/utils/google_calendar.py

```diff
@@ -62,14 +62,16 @@ def insert(self, values, token=None, timeout=TIMEOUT):
         headers = {'Content-type': 'application/json', 'Authorization': 'Bearer %s' % token}
         if not values.get('id'):
             values['id'] = uuid4().hex
-        self.google_service._do_request(url, json.dumps(values), headers, method='POST', timeout=timeout)
+        options= {'sendUpdates': 'all'}
+        self.google_service._do_request(url, json.dumps(values), headers, method='POST', timeout=timeout, options=options)
```

Comment:
This seems way more complicated than necessary, the naming is confusing and we end up with params -> data and options -> params.

In both cases the querystring parameters are *completely static*, and the URLs are string literal locals.

How about just adding the QS parameters to the URL literals directly?